app/services/open_library_service.rb

```diff
@@ -0,0 +1,20 @@
+class OpenLibraryService
+  include HTTParty
+
+  def book_info(isbn)
+    response = HTTParty.get(
+      "https://openlibrary.org/api/books?bibkeys=ISBN:#{isbn}&format=json&jscmd=data"
+    )
+    parser_json_isbn(response.parsed_response, isbn)
```

Comment:
I would name it parse instead of parser since it's an action not an actual parser